### PR TITLE
topology: sof-adl-rt1019-rt5682: add bluetooth offload uses SSP2 link

### PR DESCRIPTION
**Pull request by ajye-huang**
Adding support for  bluetooth offload

sof-adl-rt1019-rt5682:
    rt5682 headphone connects SSP0 link.
    rt1019 Amp speakers connects SSP1 link.
    bluetooth offload uses SSP2 link.

### CODE REVIEW

**Comment by sofci** [comment]
Can one of the admins verify this patch?
> reply **test&nbsp;this&nbsp;please** to run this test once

**Comment by gkbldcig** [comment]
Can one of the admins verify this patch?

**Comment by marc-hb** [comment]
test this please

File: tools/topology/topology1/CMakeLists.txt [hunk 1]
```diff
@@ -101,7 +101,7 @@ set(TPLGS
 	"sof-tgl-max98357a-rt5682\;sof-adl-max98390-rt5682-rtnr-google-aec\;-DCODEC=MAX98390\;-DFMT=s32le\;-DPLATFORM=adl\;-DLINUX_MACHINE_DRIVER=sof_rt5682\;-DAMP_SSP=1\;-DBT_OFFLOAD\;-DCHANNELS=2\;-DGOOGLE_RTC_AUDIO_PROCESSING\;-DRTNR\;-DNOHOTWORD\;-DNO16KDMIC\;-DSPK_MIC_PERIOD_US=10000"
 	"sof-tgl-max98357a-rt5682\;sof-adl-max98390-ssp2-rt5682-ssp0\;-DCODEC=MAX98390\;-DFMT=s32le\;-DPLATFORM=adl\;-DLINUX_MACHINE_DRIVER=sof_rt5682\;-DAMP_SSP=2"
 	"sof-tgl-max98357a-rt5682\;sof-adl-rt5682\;-DNO_AMP\;-DPLATFORM=adl\;-DLINUX_MACHINE_DRIVER=sof_rt5682"
-	"sof-tgl-max98357a-rt5682\;sof-adl-rt1019-rt5682\;-DCODEC=RT1019\;-DFMT=s24le\;-DPLATFORM=adl\;-DLINUX_MACHINE_DRIVER=sof_rt5682\;-DAMP_SSP=1\;-DSPK_MIC_PERIOD_US=10000"
+	"sof-tgl-max98357a-rt5682\;sof-adl-rt1019-rt5682\;-DCODEC=RT1019\;-DFMT=s24le\;-DPLATFORM=adl\;-DLINUX_MACHINE_DRIVER=sof_rt5682\;-DAMP_SSP=1\;-DSPK_MIC_PERIOD_US=10000\;-DBT_OFFLOAD"
```
Comment (plbossart) [inline tools/topology/topology1/CMakeLists.txt:104]: why would the 'waves' version below NOT have BT offload?

**Review by plbossart — CHANGES_REQUESTED** [review]

File: tools/topology/topology1/CMakeLists.txt (hunk 1 above)
Comment (johnylin76) [reply tools/topology/topology1/CMakeLists.txt:104]: Actually sof-adl-rt1019-rt5682-waves.tplg can have BT_OFFLOAD applied for sure. Although I didn't find the plan for BT offload feature bringup in the launch schedule for the project that adopts sof-adl-rt1019-rt5682-waves.tplg, The addition of BT_OFFLOAD should be harmless.

Hi Ajye, please check to add it as well.

**Review by plbossart — APPROVED** [review]

**Review by lgirdwood — APPROVED** [review]

**Comment by lgirdwood** [comment]
SOFCI TEST

**Comment by lgirdwood** [comment]
@wszypelt @lrudyX not sure why the compilation is failing here since this is a topology PR ?

**Pull request merged**